build.gradle

```diff
@@ -20,7 +20,7 @@ project.ext {
     slf4jVersion = '1.7.25'
     mockitoVersion = '1.9.5'
     assertjVersion = '3.8.0'
-    zkToolsVersion = '0.6.0'
+    zkToolsVersion = '0.7.1'
```

Comment:
Updated with new version of zktools: https://github.com/wepay/zktools/commit/c498f88cba887318320ec2ccb8f7c6b31c94d787

Reply:
@ymatsuda Can I get a review for this new change? Thanks!